### PR TITLE
Recipe for the R comorbidity package

### PR DESCRIPTION
**Pull request by abalter**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Package does not ship static libraries. If static libraries are needed, [follow CFEP-18](https://github.com/conda-forge/cfep/blob/main/cfep-18.md).
- [ ] Build number is 0. _Huh?_
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [ ] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there. _posted where?_
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by abalter** [comment]
What do I do about 
>Recipe maintainer "Ariel Balter [ariel@arielbalter.com](mailto:ariel@arielbalter.com)" does not exist

File: recipes/r-comorbidity/meta.yaml
Comment (jaimergp) [inline recipes/r-comorbidity/meta.yaml:14]: Do you need both? In that case, use:

```suggestion
    - url: {{ cran_mirror }}/src/contrib/comorbidity_{{ version }}.tar.gz
      sha256: 243d8c12901860ed5ccd5f7ec633467aff21d25164225ef8dab5ff9f06b41dd6
    - url: {{ cran_mirror }}/src/contrib/Archive/comorbidity/comorbidity_{{ version }}.tar.gz
      sha256: REPLACE_ME
```

File: recipes/r-comorbidity/meta.yaml
Comment (abalter) [reply recipes/r-comorbidity/meta.yaml:56]: Do you mean I have to have an _actual_ text file?

**Comment by jaimergp** [comment]
> What do I do about 
> >Recipe maintainer "Ariel Balter [ariel@arielbalter.com](mailto:ariel@arielbalter.com)" does not exist

Try using your github username instead.

**Comment by abalter** [comment]
@jaimergp --- Doh! Thanks. I did manage to figure that out.

File: recipes/r-comorbidity/meta.yaml
Comment (mfansler) [reply recipes/r-comorbidity/meta.yaml]: @jaimergp This is incorrect advice. For R packages with common licenses, we prefer to source them from the `r-base` as this is most consistent with how CRAN does the license distribution. Unless the package has an explicit LICENSE file, what was here was correct.

File: recipes/r-comorbidity/meta.yaml
Comment (mfansler) [inline recipes/r-comorbidity/meta.yaml:57]: R packages should include `conda-forge/r` as maintainer:

```suggestion
    - conda-forge/r
    - abalter
```

File: recipes/r-comorbidity/meta.yaml
Comment (mfansler) [inline recipes/r-comorbidity/meta.yaml:45]:
```suggestion
  home: https://ellessenne.github.io/comorbidity/
  dev_url: https://github.com/ellessenne/comorbidity/
```

File: recipes/r-comorbidity/meta.yaml
Comment (mfansler) [inline recipes/r-comorbidity/meta.yaml:25]: This should be under selector
```suggestion
    - {{ posix }}zip  # [win]
```

File: recipes/r-comorbidity/meta.yaml
Comment (mfansler) [inline recipes/r-comorbidity/meta.yaml:18]: Selector, and can be `noarch` because DESCRIPTION file shows "NeedsCompilation: no".

```suggestion
  merge_build_host: True  # [win]
  number: 0
  noarch: generic
```

File: recipes/r-comorbidity/meta.yaml
Comment (mfansler) [inline recipes/r-comorbidity/meta.yaml:12]: Revert

```suggestion
    - {{ cran_mirror }}/src/contrib/comorbidity_{{ version }}.tar.gz
    - {{ cran_mirror }}/src/contrib/Archive/comorbidity/comorbidity_{{ version }}.tar.gz
```

File: recipes/r-comorbidity/LICENSE.txt
Comment (mfansler) [inline recipes/r-comorbidity/LICENSE.txt:1]: Please remove.

File: recipes/r-comorbidity/build.sh
Comment (mfansler) [inline recipes/r-comorbidity/build.sh:22]: Remove superfluous comments

```suggestion
```

**Review by mfansler — CHANGES_REQUESTED** [review]
Thanks for submitting this recipe! There are several things to tidy up before merging.

For future submissions, note that [`conda_r_skeleton_helper`](https://github.com/bgruening/conda_r_skeleton_helper) is the preferred way to generate CRAN recipes for Conda Forge.

File: recipes/r-comorbidity/meta.yaml
Comment (abalter) [reply recipes/r-comorbidity/meta.yaml]: I removed the license.txt file and pushed.

File: recipes/r-comorbidity/LICENSE.txt
Comment (abalter) [reply recipes/r-comorbidity/LICENSE.txt:1]: Done.

**Review by mfansler — APPROVED** [review]

**Comment by mfansler** [comment]
Thanks again for adding this!

**Comment by abalter** [comment]
I'm so stoked!

**Comment by abalter** [comment]
@mfansler --- So is there another step for me to do, or will this actually show up at some point in the conda-forge channel?

**Comment by mfansler** [comment]
@abalter [the feedstock](https://github.com/conda-forge/r-comorbidity-feedstock) was just rendered, which will trigger the package to be re-built and uploaded. Usually takes a couple hours to build, upload to `cf-staging`, transfer to `conda-forge` channel, and then incorporated into the `repodata.txt`. 

Also, you should get a notification about being invited as a Conda Forge maintainer. But you can also directly head to https://github.com/orgs/conda-forge/invitation to accept.

File: recipes/r-comorbidity/meta.yaml
Comment (jaimergp) [reply recipes/r-comorbidity/meta.yaml:14]: TIL about that feature. Thanks!

**Comment by abalter** [comment]
Do I need to do something extra to get the package to show up at anaconda.org? If I search for "r comorbidity conda" I see [this](https://anaconda.org/r/r-comorbidity), but not my conda-forge build. It does show up in a search:

```
(base) balter@spectre:~$ micromamba search r-comorbidity
Getting repodata from channels...

bioconda/linux-64                                    4.6MB @   2.9MB/s  2.0s
bioconda/noarch                                      4.2MB @   2.0MB/s  2.5s
conda-forge/noarch                                  11.8MB @   3.6MB/s  3.8s
conda-forge/linux-64                                30.6MB @   4.8MB/s  8.8s


 Name          Version Build         Channel
────────────────────────────────────────────────────────
 r-comorbidity 1.0.5   r41h785f33e_0 conda-forge/noarch
 r-comorbidity 1.0.5   r42h785f33e_0 conda-forge/noarch
```

**Comment by mfansler** [comment]
It's there: https://anaconda.org/conda-forge/r-comorbidity

**Comment by abalter** [comment]
Oh! Just not showing up on google :)

**Pull request merged**
